src/components/create-safe/status/usePendingSafeCreation.ts

```diff
@@ -29,6 +31,8 @@ export const checkSafeCreationTx = async (provider: JsonRpcProvider, txHash: str
       return SafeCreationStatus.REVERTED
     }
 
+    trackEvent(CREATE_SAFE_EVENTS.CREATED_SAFE)
```

Comment:
This function should only run if the user refreshes the page after submitting the tx. Although it currently doesn't work I will create a fix for it. I suggest we track this event in the watcher when detecting `SUCCESS`.